pkg/controller/cronjob/utils_test.go

```diff
@@ -143,6 +149,16 @@ func TestGetParentUIDFromJob(t *testing.T) {
 		// Case 2: Has UID annotation
 		j.ObjectMeta.Annotations = map[string]string{v1.CreatedByAnnotation: `{"kind":"SerializedReference","apiVersion":"v1","reference":{"kind":"CronJob","namespace":"default","name":"pi","uid":"5ef034e0-1890-11e6-8935-42010af0003e","apiVersion":"extensions","resourceVersion":"427339"}}`}
 
+		j.ObjectMeta.SetOwnerReferences([]metav1.OwnerReference{
+			{
+				Kind: "CronJob",
+			},
+		})
+		j.ObjectMeta.SetName("pi")
```

Comment:
ya, the name is unnecessary. i peeked into attributes of serialized reference, and set all attributes to `j.ObjectMeta`, ensuring i have everything right in the place.